source/crud/tests/unified/aggregate-merge-readPreference.yml

```diff
@@ -0,0 +1,106 @@
+description: aggregate-merge-readPreference
+
+schemaVersion: '1.3'
+
+runOnRequirements:
+  # 3.6+ non-standalone is needed to utilize $readPreference in OP_MSG. 4.2+ is
+  # required for $merge.
```

Comment:
Requiring 4.2+ here is the only notable difference (apart from pipeline differences) between this file and `aggregate-out-readPreference.yml`.